pkg/tfvars/openstack/openstack.go

```diff
@@ -152,12 +152,12 @@ func TFVars(masterConfigs []*v1alpha1.OpenstackProviderSpec, cloud string, exter
 	}
 
 	cfg.AdditionalNetworkIDs = []string{}
-	if mpool.AdditionalNetworkIDs != nil {
+	if mpool != nil && mpool.AdditionalNetworkIDs != nil {
```

Comment:
The check that `AdditionalNetworkIDs` is not nil is unnecessary.
```suggestion
	if mpool != nil {
```